core-bundle/src/Controller/FrontendModule/RootPageDependentModulesController.php

```diff
@@ -37,8 +37,15 @@ public function __invoke(Request $request, ModuleModel $model, string $section,
             return new Response('');
         }
 
+        $moduleModel = $this->container->get('contao.framework')->getAdapter(ModuleModel::class);
+        $module = $moduleModel->findByPk($modules[$pageModel->rootId]);
+
+        $cssID = StringUtil::deserialize($module->cssID, true);
```

Comment:
The ID should get overwritten if set I think.

```suggestion
        $cssID = StringUtil::deserialize($module->cssID, true);
        if ($idAttribute = $request->attributes->get('templateProperties', [])['cssID'] ?? null) {
            $cssID[0] = substr($idAttribute, 5, -1);
        }
```

Reply:
Adjusted in 35fa17cb4f.